.gitattributes

```diff
@@ -1 +1,3 @@
 *.parquet filter=lfs diff=lfs merge=lfs -text
+*.avro filter=lfs diff=lfs merge=lfs -text
+*metadata.json filter=lfs diff=lfs merge=lfs -text
```

Comment:
I don't think it makes sense to put the json files in LFS. Typically, only larger binary files need LFS.